pkg/api/utils/apiServiceUtils.go

```diff
@@ -18,7 +18,7 @@ type APIService interface {
 	getHTTPClient() *http.Client
 }
 
-func post(uri string, data []byte, api APIService) (*models.Error, error) {
+func post(uri string, data []byte, api APIService) (*models.ChannelInfo, *models.Error) {
```

Comment:
This looks like as if it is API breaking. Will this cause any issues with the remaining project?

Reply:
I was missing the return parameter of the ChannelInfo that is required to open a WebSocket communication. No other component than the CLI is using this util functions so that there is no conflict with other components.